actions/service.go

```diff
@@ -421,3 +423,8 @@ func (s *Service) PostMergeHook(ctx context.Context, record graveler.HookRecord)
 	// post merge actions will be enabled here
 	return nil
 }
+
+func NewHookRunID(id int) string {
+	tm := time.Now().UTC().Format("02150405")
```

Comment:
You don't use the same time for the tasks, so we already have prefix which is sorted and unique, no?

Reply:
right, f2f chat - will use action and hook idx to identify the specific task

Reply:
went with 4 digits for each range - just in case.